docs/WindowsBuild.md

```diff
@@ -0,0 +1,176 @@
+# Building Swift on Windows
+
+This document describes how to build Swift for Windows natively. See [the
+Windows doc](./Windows.md) for more about what is possible with Swift on
```

Comment:
Nit: "Getting Started with Swift on Windows" is the name of that document, not "the Windows doc."